databricks/koalas/groupby.py

```diff
@@ -344,6 +344,59 @@ def any(self):
             lambda col: F.max(F.coalesce(col.cast('boolean'), F.lit(False))),
             only_numeric=False)
 
+    # TODO: groupby multiply columuns should be implemented.
+    def size(self):
+        """
+        Compute group sizes.
+
+        See Also
+        --------
+        databricks.koalas.Series.groupby
+        databricks.koalas.DataFrame.groupby
+
+        Examples
+        --------
+        >>> df = ks.DataFrame({'A': [1, 2, 2, 3, 3, 3],
+        ...                    'B': [1, 1, 2, 3, 3, 3]},
+        ...                   columns=['A', 'B'])
+        >>> df
+           A  B
+        0  1  1
+        1  2  1
+        2  2  2
+        3  3  3
+        4  3  3
+        5  3  3
+
+        >>> df.groupby('A').size()  # doctest: +NORMALIZE_WHITESPACE
+        A
+        1  1
+        2  2
+        3  3
+        Name: count, dtype: int64
+
+        >>> df.groupby(['A', 'B']).size()  # doctest: +NORMALIZE_WHITESPACE
+        A  B
+        1  1    1
+        2  1    1
+           2    1
+        3  3    3
+        Name: count, dtype: int64
+        """
+        groupkeys = self._groupkeys
+        groupkey_cols = [s._scol.alias('__index_level_{}__'.format(i))
+                         for i, s in enumerate(groupkeys)]
+        sdf = self._kdf._sdf
+        data_columns = ['count']
+        sdf = sdf.groupby(*groupkey_cols).count()
+        sdf = sdf.sort(*groupkey_cols)
```

Comment:
@RainFung, it's okay not to guarantee the order for now. Let's don't sort if that's the purpose.